Dockerfile

```diff
@@ -1,15 +1,25 @@
 FROM ruby:2.4-alpine
-MAINTAINER Justin Collins
+LABEL maintainer="Justin Collins"
```

Comment:
@presidentbeef not sure if you wanna give this level of detail, but normally the Maintainer would come with some form of a contact method, e.g. email address.

Reply:
Thanks, I will update that after merge.